pgmgr/pgmgr_test.go

```diff
@@ -257,6 +257,20 @@ func TestMigrate(t *testing.T) {
 
 	psqlMustExec(t, `SELECT * FROM bars;`)
 
+	// Make a filename that would match a vim .swp file
+	writeMigration(t, ".003_this_is_an_older_migration.up.sql.swp", `
+		CREATE TABLE baz (baz_id INTEGER);
+		INSERT INTO baz (baz_id) VALUES (4), (5), (6);
+	`)
+
+	err = Migrate(globalConfig())
+	if err != nil {
+		t.Log(err)
+		t.Fatal("Could not apply third migration!")
```

Comment:
Since the third migration isn't really a migration, we don't expect it to even run. So, the error message here is a bit confusing. How about something like `Migration returned an error instead of being skipped`?